.github/test-spec.yml

```diff
@@ -180,3 +180,9 @@
       - "crypto/**/*"
       - "!crypto/doc/**/*"
       - "!crypto/*.rst"
+
+"CI-audio-test":
```

Comment:
@Frodevan : Is there a way CI can run with this controller when changes are pushed to this repo?

Reply:
This has now been fixed for CI